testsuite/tests/libratest/smoke_test.rs

```diff
@@ -52,7 +52,7 @@ impl TestEnvironment {
         ::libra_logger::Logger::new().init();
         let mut template = NodeConfig::default();
         template.test = Some(TestConfig::open_module());
-        template.state_sync.chunk_limit = 2;
+        template.state_sync.chunk_limit = 5;
```

Comment:
context?

Reply:
oh it's a previous PR to avoid flakiness in smoke tests, just need rebase